fe/src/components/Modal.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import ReactDOM from 'react-dom';
+
+const Modal = ({
+  onClose, children, actionBar,
+}) => {
+  const handleKeyDown = (event) => {
+    if (event.key === 'Enter') {
+      document.body.classList.remove('overflow--hidden');
+    }
+  };
+  useEffect(() => {
+    document.body.classList.add('overflow-hidden');
+    return () => {
+      document.body.classList.remove('overflow--hidden');
+    };
+  }, []);
```

Comment:
- insert new line in each functions closing tag. add newline after **line:11** (apply to all related)